1차/이동건_모의고사.py

```diff
@@ -0,0 +1,30 @@
+# 20:55 ~ 21:03
+
+def createWay(arr):
+    def generator():
+        i = 0
+        while 1:
+            yield arr[i%len(arr)]
+            i = i + 1
```

Comment:
1. 내장함수의 의도를 모르겠습니다.
```suggestion
def createWay(arr):
      i = 0 
      while True:
          yield arr[i%len(arr)]
          i = i+1
```